demo/app-root.ts

```diff
@@ -17,26 +17,46 @@ import { ModalManager } from '@internetarchive/modal-manager';
 import '@internetarchive/modal-manager';
 import { ItemNavigator } from '../src/item-navigator';
 import '../src/item-navigator';
+import {
+  MenuShortcutInterface,
+  MenuDetailsInterface,
+} from '../src/interfaces/menu-interfaces';
 @customElement('app-root')
 export class AppRoot extends LitElement {
   /**
    * Example controller to connect to `<ia-item-navigator>`
    */
-  @property({ type: Object }) itemMD: MetadataResponse | undefined = undefined;
+  @property({ type: Object }) itemMD?: MetadataResponse;
 
   @property({ type: String }) encodedManifest = '';
 
-  @query('ia-item-navigator') private itemNav!: ItemNavigator;
+  @property({ attribute: false }) sharedObserver = new SharedResizeObserver();
 
-  @query('modal-manager') modalMgr!: ModalManager;
+  @property({ type: Array, attribute: false })
+  menuContents: MenuDetailsInterface[] = [];
 
-  @property({ attribute: false }) sharedObserver = new SharedResizeObserver();
+  @property({ type: Array, attribute: false })
+  menuShortcuts: MenuShortcutInterface[] = [];
 
-  @property({ type: Boolean, reflect: true, attribute: true }) fullscreen:
+  @property({ reflect: true, attribute: true }) fullscreen:
     | boolean
     | null = null;
 
-  @property({ type: Boolean, reflect: true, attribute: true }) headerOn = false;
+  @property({ reflect: true, attribute: true }) headerOn: true | null = null;
```

Comment:
why do we change `headerOn` value to be null instead of `true` or `false`?

same with the others, `showPlaceholder` and `showTheaterExample`?

Reply:
hi @dualcnhq - great question -  it's a lit element property convention where, if one wants a property to be reflected back to the main node as an attribute, it needs `true` to add it, and `null` to remove it.

this is for the demo and its properties. it also helps me easily debug just by looking at the elements in dev tool ^_^